partnercenter/azext_partnercenter/models/resource.py

```diff
@@ -0,0 +1,18 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from msrest.serialization import Model
+
+
+class Resource(Model):
```

Comment:
This represents the "resource" that can be on all our models so that "system" id and type can be differentiated from the ID attr that will be on the models, which the consumer will be aware of.